src/pages/ReplaceServer/PrepareReplace.js

```diff
@@ -31,6 +31,7 @@ const PLAYBOOK_STEPS = [
     label: translate('deploy.progress.ready-deployment'),
     playbooks: ['ready-deployment.yml']
   },
+  // TODO: Remove this. It is just confusing
```

Comment:
It's not clear what this comment in referring to

Reply:
something I removed but forgot to remove the comment. Will update

Reply:
actually, it's a valid comment that needs clarification. This code is confusing in that it runs the config processor playbook, then ready deployment, then the installui-pre-deployment (PRE_DEPLOYMENT_PLAYBOOK) -- the confusing part is that this last playbook just runs those first two playbooks again.

Reply:
I think there is some code somewhere that causes them to be run once instead of twice, but it is really confusing to define essentially the same set of playbooks twice.

Reply:
Added a commit to clarify that comment